include/seqan3/utility/math.hpp

```diff
@@ -12,11 +12,13 @@
 
 #pragma once
 
+#include <seqan3/std/bit>
+#include <cassert>
```

Comment:
Do we need cassert or is this a left over from debugging?

Reply:
See below